### ISSUE TITLE
Resume training

### ISSUE THREAD

**Issue by dorpxam** [LongCatFan1/longcat-tuner#14]
There's two way to restart (resume) a training using `musubi-tuner`:

- `--network_weights` : single `.safetensors` that initialize the weights (need to fit LoRA rank)
- `--resume` : resume the training from the selected directory saved previously by `--save_state`

I notice that even if `--resume` use the state, the epoch counter reset to 1. 

So, for example, if I have a 64 epochs training settings, train 32 epochs, and attempt to resume the training, the epoch needs to restart at 33/64, not 1/64.

Anyway, it's not really a problem of training, this is more a cosmetic issue to avoid this kind of loss graph:

<img width="2224" height="1278" alt="Image" src="https://github.com/user-attachments/assets/41f82092-73a6-4844-9186-d2c19e75ced1" />
